src/commands/codepush/release-react.ts

```diff
@@ -48,6 +48,12 @@ export default class CodePushReleaseReactCommand extends CodePushReleaseCommandB
   @hasArg
   public plistFilePrefix: string;
 
+  @help("Name of build configuration which specifies the binary version you want to target this release at. For example, \"Debug\" or \"Release\" (iOS only)")
+  @shortName("c")
+  @hasArg
+  @longName("build-configuration-name")
+  public buildConfigurationName: string;
```

Comment:
Is this a mandatory argument?

Reply:
Oh I just saw in another line that it defaults to Release. Why not here?

Reply:
Yes, you're right. I already fixed it. Thanks!